compiler/front/nimconf.nim

```diff
@@ -438,9 +435,7 @@ proc loadConfigs(
 
   proc runNimScriptIfExists(N: var NimConfParser, path: AbsoluteFile) =
     if fileExists(path):
-      var s = llStreamOpen(path, fmRead)
-      N.config.configFiles.add(path)
-      runNimScript(cache, path, freshDefines = false, N.config, s)
+      doAssert false, "nimscript for config no longer supported: " & $path
```

Comment:
way to detect `config.nims` and what not, going to simply drop this whole proc and all calls to it.